projects/sidechain-connector-service-typescript/src/types/web3.ts

```diff
@@ -0,0 +1,401 @@
+import { BigNumber } from "bignumber.js"; // TODO change to BN
+import * as us from "underscore";
```

Comment:
What about using `lodash` instead? It's better and more optimized than `underscore` and AFAIK, we're already using it elsewhere.

Reply:
it's an external lib. I haven't touched it